docs/_tutorials/scenarios_suites/HuggingFace.rst

```diff
@@ -0,0 +1,70 @@
+HuggingFace's NLP datasets
+----------------------------
+
+
+HuggingFace proposes the largest amount of NLP datasets. The full list
+can be found `here <https://huggingface.co/docs/datasets/>__`.
+
+Naturally, Continuum can work with those NLP datasets and convert them in
+continual scenarios.
+
+In all cases, you won't work with `TaskSet` but instead directly with HuggingFace's Dataset.
```

Comment:
This is not very clear what this means for someone not knowing well continuum.
I would rather say something like "Continuum scenarios create object that iterates tasks, in classifcation those tasks are implemented as `Taskset` that con be loaded with pytorch as [...], with NLP the task are not pytorch loader object but huggingFace objects "

Reply:
right, good call